README.md

```diff
@@ -34,6 +34,29 @@ make test
 See [`CONTRIBUTING.md`](CONTRIBUTING.md) for details on how to contribute to
 this project.
 
+The code in this project is formatted with `clang-format(1)`, and our CI builds
+will check that the code matches the format generated by this tool before
+accepting a pull request.
+Please configure your editor or IDE to use the Google style for indentation and
+whitespace.
+If you need to reformat one or more files you can simply:
+
+```console
+$ clang-format -i <file>....
+```
+
+Reformatting all the files in a specific directory should be safe too, for 
+example:
+
+```console
+$ clang-format -i bigtable/client/*.h bigtable/client/*.cc
+```
+
+If you need to reformat one of the files to match the Google style.
```

Comment:
Please reflow this paragraph as well.

Reply:
Done.  Tough they render the same and I find them easier to edit when not reflowed, meh.